apps/src/code-studio/pd/workshop_dashboard/components/workshop_table.jsx

```diff
@@ -281,7 +280,9 @@ export default class WorkshopTable extends React.Component {
         date={date}
         editUrl={state === 'Not Started' ? `/workshops/${id}/edit` : null}
         onDelete={state !== 'Ended' ? this.props.onDelete : null}
-        showSurveyUrl={state === 'Ended' || [WorkshopTypes.local_summer, WorkshopTypes.teachercon].includes(subject)}
+        showSurveyUrl={state === 'Ended' || (
+          ['CS Discoveries', 'CS Principles'].includes(course) && subject !== 'Code.org Facilitator Weekend'
+        )}
```

Comment:
same here. Can this be DRY'd up with the above, or better moved into the server / model?